app/javascript/hyacinth_ui_v1/components/ui/buttons/EditButton.js

```diff
@@ -16,6 +16,7 @@ class EditButton extends React.PureComponent {
           {...rest}
         >
           <FontAwesomeIcon icon="pen" />
+          { children }
```

Comment:
is children a list of components?

Reply:
it can be, but it doesn't have to be. I added this because I wanted to add 'Edit' after the icon.